storage.sound-editing.uploads.rules

```diff
@@ -0,0 +1,9 @@
+service firebase.storage {
+  match /b/{bucket}/o {
+    match /{uploadCode}/{file=**} {
+      allow write: if request.resource.contentType.matches('audio/flac') &&
+          request.resource.name.matches('.*\\.flac');
```

Comment:
Previously you said that you cannot use regex here. Isn't it a regex? If it is then why not to test for the task ID pattern as well?